docs/sources/k6/v0.57.x/shared/blocking-aws-blockquote.md

```diff
@@ -7,6 +7,6 @@ title: jslib/aws module blocking admonition
 In some cases, using this library&apos;s operations might impact performance and skew your test results.
 <br>
 <br>
-To ensure accurate results, consider executing these operations in the `setup` and `teardown` [lifecycle functions]({{< relref "../using-k6/test-lifecycle" >}}). These functions run before and after the test run and have no impact on the test results.
+To ensure accurate results, consider executing these operations in the `setup` and `teardown` [lifecycle functions](../../using-k6/test-lifecycle/). These functions run before and after the test run and have no impact on the test results.
```

Comment:
```suggestion
To ensure accurate results, consider executing these operations in the `setup` and `teardown` [lifecycle functions](htts://grafana.com/docs/k6/<K6_VERSION>/using-k6/test-lifecycle/). These functions run before and after the test run and have no impact on the test results.
```

I tried to run this locally and I think the link isn't working. :( Maybe we just replace it with the full URL?